packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -45,7 +45,7 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    * Whether the ComboBox auto completes. As the user is inputing text, it will be suggested potential matches from the list of items. If
    * the combo box is expanded, this will also scroll to the suggested item, and give it a selected style. Defaults to false.
    */
-  autoComplete?: boolean;
+  autoComplete?: string;

```

Comment:
Given that the component was just introduced, it is probably safe to make this change. Other scenarios are broken because the props don't subclass from HTMLAttributes.